gensim/models/doc2vec.py

```diff
@@ -286,6 +286,9 @@ def __init__(self, documents=None, corpus_file=None, vector_size=100, dm_mean=No
 
         self.vector_size = vector_size
         self.dv = dv or KeyedVectors(self.vector_size, mapfile_path=dv_mapfile)
+        # EXPERIMENTAL lockf feature; create minimal no-op lockf arrays (1 element of 1.0)
+        # advanced users should directly resize/adjust as desired after any vocab growth
```

Comment:
Do we have any other documentation for such advanced users, or is this it?

Reply:
This is it! It's only ever existed as an advanced experiment most users should ignore, and only users comfortable using the source code as documentation should consider using.